tools/pd-ctl/tests/global_test.go

```diff
@@ -16,34 +16,51 @@ package tests
 
 import (
 	"context"
-	"fmt"
+	"encoding/json"
 	"net/http"
 	"testing"
 
+	"github.com/pingcap/kvproto/pkg/metapb"
 	"github.com/pingcap/log"
 	"github.com/stretchr/testify/require"
 	"github.com/tikv/pd/pkg/utils/apiutil"
 	"github.com/tikv/pd/pkg/utils/assertutil"
 	"github.com/tikv/pd/pkg/utils/testutil"
 	"github.com/tikv/pd/server"
+	"github.com/tikv/pd/server/api"
 	cmd "github.com/tikv/pd/tools/pd-ctl/pdctl"
 	"go.uber.org/zap"
 )
 
-const pdControlCallerID = "pd-ctl"
+const pdControlCallerID = "pd-http-client"
```

Comment:
there need to be `pd-ctl`, maybe You can reuse the variables in the `global.go` file directly

https://github.com/tikv/pd/blob/7e18a69734f2ae391516625ec0230c05b363f5c5/tools/pd-ctl/pdctl/command/global.go#L36

FYI
https://github.com/tikv/pd/blob/7e18a69734f2ae391516625ec0230c05b363f5c5/tools/pd-ctl/pdctl/command/global.go#L103-L112

Reply:
It's a little strange here. The unit test requires it to be `pd-http-client`. I'll do some research for it

Reply:
https://github.com/tikv/pd/blob/4cd42b3cb8e7ae32c4d15d6fab7d7ab0552facf2/client/http/client.go#L310

The HTTP client created in the `pd-ctl` is assigned by `defaultCallerID`. Do we need to change it to `source`. `source` equals to "pd-ctl" string.

Reply:
It is a bug :( Have been fixed in this pr. https://github.com/tikv/pd/pull/8214

Reply:
I think you don't need open a new pr to fix it. We even need to modify the conflict for it. If it is bug, I can fix it in this pr easily. How was my propose?

Reply:
I think this is a different issue, maybe the new pr will be a little clearer 0 0

Reply:
It is ok for me